.github/workflows/android-check-metadata.yaml

```
@@ -15,6 +15,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          sparse-checkout: |
+            android/app/src/fdroid
+            android/app/src/google
```

Let's make it more generic.
```suggestion
            android
```

---

done